src/config.ts

```diff
@@ -3,3 +3,12 @@ export const dappConfig = {
   INSTANCE_PREFIX: ':published.psm.IST.',
   INSTANCES_KEY: ':published.agoricNames.instance',
 };
+
+export const wellKnownBrands = {
```

Comment:
we could also derive that other list of tokens from this one.

Reply:
yeah, though it's different for mainnet and tests, right? a different set